tests/unit/strict-resolver/basic-test.js

```diff
@@ -135,3 +137,148 @@ test("can lookup a view in another namespace with different syntax", function(as
   assert.ok(view, 'view was returned');
   assert.equal(view, expected, 'default export was returned');
 });
+
+test("can lookup a view", function(assert) {
```

Comment:
Yeah, I figured.  Seems like we should have started by forking that repo.  Its _slightly_ annoying to loose all the proper attributions and bug history / reasoning.